From 53bc50c903bffad28c1d6098d983988ff2f20ac9 Mon Sep 17 00:00:00 2001
From: Saahil Tomar <quic_saahtoma@quicinc.com>
Date: Wed, 31 May 2023 09:53:51 +0530
Subject: [PATCH] arm64: dts: qcom: ipq9574: Add build support for RDPs

Adding build support for the following RDPs
RDP 433-mht-switch,
RDP 437,
RDP 461

Change-Id: I3ffda26c165497fdc12c03fe0bdaebd5cddc2f85
Signed-off-by: Saahil Tomar <quic_saahtoma@quicinc.com>
---
 arch/arm/boot/dts/Makefile        | 3 +++
 arch/arm64/boot/dts/qcom/Makefile | 3 +++
 2 files changed, 6 insertions(+)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index 076a060ae149..50d2cb77dfac 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -1052,6 +1052,8 @@ dtb-$(CONFIG_ARCH_QCOM) += \
 	ipq9574-rdp418.dtb \
 	ipq9574-rdp433.dtb \
 	ipq9574-rdp433-mht-phy.dtb \
+	ipq9574-rdp433-mht-switch.dtb \
+	ipq9574-rdp437.dtb \
 	ipq9574-rdp449.dtb \
 	ipq9574-rdp453.dtb \
 	ipq9574-rdp454.dtb \
@@ -1060,6 +1062,7 @@ dtb-$(CONFIG_ARCH_QCOM) += \
 	ipq9574-rdp457.dtb \
 	ipq9574-rdp458.dtb \
 	ipq9574-rdp459.dtb \
+	ipq9574-rdp461.dtb \
 	ipq9574-rdp467.dtb \
 	ipq9574-rdp469.dtb \
 	ipq5332-rdp441.dtb \
diff --git a/arch/arm64/boot/dts/qcom/Makefile b/arch/arm64/boot/dts/qcom/Makefile
index 796682457fc0..8939020fdd70 100644
--- a/arch/arm64/boot/dts/qcom/Makefile
+++ b/arch/arm64/boot/dts/qcom/Makefile
@@ -16,6 +16,8 @@ dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp417.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp418.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp433.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp433-mht-phy.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp433-mht-switch.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp437.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp449.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp453.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp454.dtb
@@ -24,6 +26,7 @@ dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp456.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp457.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp458.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp459.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp461.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp467.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq9574-rdp469.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= msm8916-alcatel-idol347.dtb
-- 
2.34.1

